convene-web/spec/requests/spaces/invitations/rsvp_spec.rb

```diff
@@ -0,0 +1,27 @@
+RSpec.describe "/spaces/:space_id/invitations/:invitation_id/rsvp", type: :request do
+  describe "GET /spaces/:space_id/invitations/:invitation_id/rsvp" do
+    context 'as a guest' do
+    end
+
+    context 'as a neighbor' do
+    end
+
+    context 'as a space member' do
+    end
```

Comment:
Probably need to add a `@todo` to flesh out this more fully so that someone else can grab it after we merge.